### PR TITLE
Dashboard update

### PR DESCRIPTION
**Pull request by designhorf**
Changes: 

New layout for Dashboard

- [x] My Tokens widget design
- [x] My Tokens functions
- [x] Buy KEY widget design
- [x] Buy KEY popup
- [x] Transactions widget design
- [x] Empty Marketplace Application route to marketplace
- [x] Marketplace Application widget design
- [x] Marketplace Application fetch data
- [x] Marketplace Application route to applications
- [x] Empty SK Profile widget
- [x] SK Profile widget design
- [x] SK Profile w/ real data
- [x] SK Profile w/ route

This PR doesn't contain the following features: 
- Transactions filter popup - Separate task - https://github.com/SelfKeyFoundation/Identity-Wallet/issues/1979
- Update Send Custom Token modal - Separate task - https://github.com/SelfKeyFoundation/Identity-Wallet/issues/2036

### CODE REVIEW

File: src/renderer/dashboard/dashboard-marketplace-applications.jsx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+					/>
+				)}
+			</Grid>
+		);
+	}
+}
+
+const mapStateToProps = state => {
+	return {
+		applications: kycSelectors.selectApplications(state)
```
Comment (andregoncalves) [inline src/renderer/dashboard/dashboard-marketplace-applications.jsx]: I'm not sure about this, I think this needs a 
`await dispatch(kycOperations.loadApplicationsOperation()); `
to show the applications

Comment (designhorf) [reply src/renderer/dashboard/dashboard-marketplace-applications.jsx]: Good point.

**Comment by andregoncalves** [comment]
@designhorf merge conflict

**Comment by designhorf** [comment]
> @designhorf merge conflict

will fix it

**Pull request merged**